packages/serialization/form/src/formParseNode.ts

```diff
@@ -78,7 +81,9 @@ export class FormParseNode implements ParseNode {
   public getObjectValue = <T extends Parsable>(
     parsableFactory: ParsableFactory<T>,
   ): T => {
-    const value: T = {} as T;
+    const temp: T = {} as T;
+    const enableBackingStore = isBackingStoreEnabled(parsableFactory(this)(temp));
+    const value: T = enableBackingStore ? new Proxy({}, createBackedModelProxyHandler<T>()) as T : temp;
```

Comment:
could we just not pass "temp" into the proxy first argument as we create the proxy? so we avoid creating yet another object?